### PR TITLE
check if SpaceBinding is being deleted

### PR DESCRIPTION
**Pull request by MatousJobanek**
https://redhat-internal.slack.com/archives/CHK0J6HT6/p1682629533095989

### CODE REVIEW

**Review by alexeykazakov — APPROVED** [review]

**Review by ranakan19 — APPROVED** [review]
Thank You!

File: testsupport/wait/host.go [hunk 1]
```diff
@@ -2276,8 +2279,14 @@ func (a *HostAwaitility) CreateSpaceAndSpaceBinding(t *testing.T, mur *toolchain
 			return false, err
 		}
 		if spaceBinding == nil {
+			t.Logf("The created SpaceBinding %s is not present", spaceCreated.Name)
 			return false, nil
 		}
+		if util.IsBeingDeleted(spaceCreated) {
```
Comment (mfrancisc) [inline testsupport/wait/host.go:2285]:
```suggestion
		if util.IsBeingDeleted(spaceBinding) {
```

File: testsupport/wait/host.go [hunk 2]
```diff
@@ -2276,8 +2279,14 @@ func (a *HostAwaitility) CreateSpaceAndSpaceBinding(t *testing.T, mur *toolchain
...
+			// spacebinding is in terminating let's wait until is gone and recreate it ...
+			t.Logf("The created SpaceBinding %s is being deleted", spaceCreated.Name)
```
Comment (mfrancisc) [inline testsupport/wait/host.go:2287]:
```suggestion
			t.Logf("The created SpaceBinding %s is being deleted", spaceBinding.Name)
```

**Review by mfrancisc — COMMENTED** [review]
Thanks! 

Just few changes suggested , If I well understood the logic.

**Review by mfrancisc — APPROVED** [review]
👍

**Pull request merged**